static/app/views/issueList/issueViews/issueViewSaveButton.tsx

```diff
@@ -121,7 +121,7 @@ export function IssueViewSaveButton({query, sort}: IssueViewSaveButtonProps) {
     openModal(props => (
       <CreateIssueViewModal
         {...props}
-        name={view?.name}
+        name={view ? `${view.name} (Copy)` : 'New View'}
```

Comment:
Could go either way but I'd prefer the input to be empty rather than "New View" if saving from nothing